src/openrct2/peep/Guest.cpp

```diff
@@ -5113,7 +5113,7 @@ void Guest::UpdateRideShopLeave()
             return;
     }
 
-    SetState(PEEP_STATE_WALKING);
+    SetState(PEEP_STATE_FALLING);
```

Comment:
Put a comment above this saying it used to be `PEEP_STATE_WALKING`, and why it was changed. Link to the issue as well.

Especially for the black magic that is pathfinding, I prefer if changes are properly documented. :-)

Reply:
Updated. At this line in the pathfinding, Peeps are at the exit point of the shop/ride. It isn't a perfect solution to set them to falling, but it does prevent them from walking until they've truly exited the shop/ride.